astropy/units/vounit_needed.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This package defines units required by the VOUnit standard but that are likely
```

Comment:
Maybe there needs to be a note (can be a comment line if you want) to explain why this could not be in `deprecated`, so people don't attempt that in the future.